content/1.2/docs/serverless-workflows/assessment/mta-analysis.md

```diff
@@ -3,4 +3,4 @@ title: MTA Analysis
 date: "2024-02-29"
 ---
 
-{{< remoteMD "https://github.com/parodos-dev/serverless-workflows/blob/main/mta/README.md?raw=true" >}}
+{{< remoteMD "https://github.com/parodos-dev/serverless-workflows/blob/main/mta-v7.x/README.md?raw=true" >}}
```

Comment:
let's point to the respected 1.2 branch:

https://raw.githubusercontent.com/parodos-dev/serverless-workflows/refs/heads/v1.2.x/mta-v7.x/README.md

but pls verify the target contains the correct content. we might need to backport some content.

Reply:
And the backport should be done https://raw.githubusercontent.com/parodos-dev/serverless-workflows/refs/heads/v1.2.x/mta-v7.x/README.md anyway no?

Reply:
by "some content" I was referring to differences between the README of the two branches. I noticed there are differences between the main to v1.2.x branches which might happen due updates pushed to the main after it was branched.